<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loadcell_main</class>
 <widget class="QMainWindow" name="loadcell_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../AEL form/AEL-removebg-preview.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>称重传感器校准模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>称重传感器#1值 ：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>称重传感器#2值 ：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>平均 ：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_loadcell_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>159</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>进行校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_home">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_loadcell_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>330</y>
      <width>159</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>归零调整</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
